### PR TITLE
Fixes #22711 - Enable ES6 support for uglifier

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: bundler.d/assets.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@
 gem 'gridster-rails', '~> 0.5'
 gem 'gettext_i18n_rails_js', '~> 1.0'
 gem 'execjs', '>= 1.4.0', '< 3.0'
-gem 'uglifier', '>= 1.0.3'
+gem 'uglifier', '>= 4.0.0'
```
Comment (ekohl) [inline bundler.d/assets.rb:7]: Our 5.1 SCL contains 3.2.0 so that'd need a bump.

Comment (mmoll) [reply bundler.d/assets.rb:7]: https://github.com/theforeman/tfm-ror51-packaging/pull/44

**Comment by mmoll** [comment]
hrm, while it seems to progress further, the build stills fails at the second webpack... no idea what's going on...

**Comment by mmoll** [comment]
I do think now, this does not fix the issue, as the problem is in the uglifier that's used by webpack and unrelated to the Rails part of this.

**Pull request closed without merging**